samples/csharp/MicrosoftEduImportFromGoogle/Export.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Microsoft.Extensions.Configuration;
```

Comment:
Can we add a brief description on what this file does, and also a one line description for each method?

Reply:
done